database/sql/src/main/scala/cromwell/database/sql/CallCachingSqlDatabase.scala

```diff
@@ -15,5 +16,5 @@ trait CallCachingSqlDatabase {
                       (implicit ec: ExecutionContext): Future[Option[CallCachingJoin]]
 
   def invalidateCall(callCachingEntryId: Int)
-                    (implicit ec: ExecutionContext): Future[Unit]
+                    (implicit ec: ExecutionContext): Future[Option[CallCachingEntry]]
```

Comment:
@mcovarr At some point you were looking at monad transformers to "unsuck" our `Future[Option]`s (`https://www.youtube.com/watch?v=hGMndafDcc8`) and I know we dropped it due to something or other. What was the something or other?